qdrant-landing/content/documentation/fastembed/fastembed-semantic-search.md

```diff
@@ -35,36 +35,68 @@ metadata = [
 ]
 ids = [42, 2]
 ```
-## Load data to a collection
-Create a test collection and upsert your two documents to it. 
+## Create a collection
+
+Qdrant stores vectors and associated metadata in collections.
+Collection requires vector parameters to be set during creation.
+In this tutorial, we'll be using `BAAI/bge-small-en` to compute embeddings.
+
 ```python
-client.add(
+client.create_collection(
     collection_name="test_collection",
-    documents=docs,
-    metadata=metadata,
-    ids=ids
+    vectors_config=models.VectorParams(size=384, distance=models.Distance.COSINE)  # size and distance are model dependent, however, distance is almost always cosine
```

Comment:
Let's use function to get embeddign size, and pls make snippet shorter, so it renders without scroll